lib/src/imap/mailbox.dart

```diff
@@ -86,9 +88,22 @@ class Mailbox {
     this.extendedData = const {},
   })  : name = _modifiedUtf7Codec.decodeText(encodedName),
         path = _modifiedUtf7Codec.decodeText(encodedPath) {
+    log('dola from enoughMail ${name.toLowerCase()}');
```

Comment:
This log does not seem to be that useful

Reply:
ok